docs/source/quick_start.rst

```diff
@@ -0,0 +1,53 @@
+.. _quick_start:
+
+Quick start
+===========
+
+Sample data
+-----------
+
+*xdem* comes with some sample data that is used throughout this documentation to demonstrate the features. If not done already, the sample data can be downloaded with the command
+
+.. code-block:: python
+
+        xdem.examples.download_longyearbyen_examples(overwrite=False)
```

Comment:
Overwrite is implicitly `False`. Honestly it took me a while to even figure out what that was, so I think it's best to remove here!

Reply:
+1

Reply:
Done